src/pages/HomePage/pages/MarketPlace/components/forms/ProductForm.tsx

```diff
@@ -48,11 +33,9 @@ export function ProductForm({
       }}
     >
       {({ values, setFieldValue, handleSubmit, errors, touched }) => {
-        const selectedSizes: string[] = values.sizes || [];
-
         return (
           <Form className="px-5 space-y-6">
-            <div className="grid grid-cols-1 lg:grid-cols-3 gap-6">
+            <div className="g">
               <div className="lg:col-span-2 space-y-4">
```

Comment:
and this div as well 
is something breaking without it 
let FormLayout handle the layout

Reply:
there is the FullWidth if you need it